### PR TITLE
ShimmedDetailsList: export missing APIs to the surface

### PR DESCRIPTION
**Pull request by Vitalius1**
#### Pull request checklist

- [X] Include a change request file using `$ npm run change`

#### Description of changes

Exporting to the surface the missing API items for `ShimmeredDetailsList`. Also, changed the `docCategory` tag to `DetailsList`.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9087)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.types.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { IStyleFunctionOrObject } from '../../Utilities';
 
 /**
  * ShimmeredDetailsList props interface
- * {@docCategory ShimmeredDetailsList}
+ * {@docCategory DetailsList}
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.types.ts]: @natalieethell , @KevinTCoughlin I have changed the `docCategory` tag to `DetailsList`. I think it's more appropriate to have it on the DetailsList page than on Reference page.

Comment (KevinTCoughlin) [reply packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.types.ts]: Seems reasonable, but even `DetailsList` is a bit of a catch-all at this point. Is there any way or usefulness to get these interfaces on https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/shimmer?

Comment (Vitalius1) [reply packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.types.ts]: I don't think we are placing any docs on subpages. @natalieethell can you confirm? Is there a way to easily add them to that page? Now it's either `DetailsList` or `References`. So where should we place it?

Comment (KevinTCoughlin) [reply packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.types.ts]: `DetailsList` is fine if its not easy.

Comment (natalieethell) [reply packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.types.ts]: Ah, yeah with the current implementation, it's not possible to place docs on subpages—although I could see the potential need for that. I think placing it under `DetailsList` makes sense to me!

**Pull request merged**